### ISSUE TITLE
native ads overlay the banner Ad

### ISSUE THREAD

**Issue by bx67212** [admob-plus/admob-plus#677]
Is there a way to set z-index?

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
Yeah, it overlays everything, "even" fixed positioned elements of my GUI.

It looks like it's an iFrame which you can't control, but maybe I am wrong.

It also slows down scrolling, so I made a script that hides it, when it's out of viewport view.

But it's nice to have, you can place an ad in your content, which is nice.

**Comment by bx67212** [comment]
Oh no, is a native view but I think @ratson can give to banner another priority

**Comment by 1n3JgKl9pQ6cUMrW** [comment]
It's a native ad, which object is wrapped in side an iFrame, I think.

When inspecting, you can "even" see the protocol and page of the ad, which traces back to googleads.g.doubleclick.net/mads/static/mad/sdk/native/production/native_ads.html

So basically it's layered on top of the webview where Cordova takes place.

You can't reach it, since it's outside the scope of Cordova.
